src/common/config_parser.cpp

```diff
@@ -562,7 +562,13 @@ void ConfigParser::addOptionsValidation(cli::CLIWrapper& cli) {
       "Additional args passed to --valid-script-path. These are inserted"
       " between the script path and the output translation-file path");
   cli.add<std::string>("--valid-translation-output",
-     "Path to store the translation");
+     "(Template for) path to store the translation\n"
+     "    template parameters:\n"
+     "    {E} for epoch\n"
+     "    {B} for # of batches within epoch\n"
+     "    {U} for total # of updates\n"
+     "    {T} for total # of tokens seen\n"
+     "    e.g., valid-after-{U}-updates-{T}-tokens.out");
```

Comment:
Unfortunately, the CLI library doesn't seem to support multi-line help messages, so everything will be aligned to the left. I think a one-liner is better in that case

Reply:
Done.